client/sse.go

```diff
@@ -305,7 +305,8 @@ func (c *SSEMCPClient) sendRequest(
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK &&
-		resp.StatusCode != http.StatusAccepted {
+		resp.StatusCode != http.StatusAccepted &&
```

Comment:
The sse handling has been moved to a separate transport in the `transport` dir. Could you pull in the latest changes and add your fix here https://github.com/mark3labs/mcp-go/blob/b46fa54c4c9404f59eb4762c027810815a88ad71/client/transport/sse.go#L284

Reply:
ok, I will fix it